server/lib/src/idm/oauth2.rs

```diff
@@ -502,15 +502,15 @@ impl<'a> IdmServerProxyWriteTransaction<'a> {
 
         // Get the o2rs for the handle.
         let o2rs = self.oauth2rs.inner.rs_set.get(&client_id).ok_or_else(|| {
-            admin_warn!("Invalid oauth2 client_id");
+            admin_warn!("Invalid OAuth2 client_id");
             Oauth2Error::AuthenticationRequired
         })?;
 
         // check the secret.
         match &o2rs.type_ {
             OauthRSType::Basic { authz_secret, .. } => {
                 if authz_secret != &secret {
-                    security_info!("Invalid oauth2 client_id secret");
+                    security_info!("Invalid OAuth2 client_id secret");
```

Comment:
While we're looking, let's add "This can happen if you were expecting a public client" or similar, see chat today.